inc/group_user.class.php

```diff
@@ -148,30 +148,21 @@ static function showForUser(User $user) {
          echo "<tr class='tab_bg_2'><td class='center'>";
          echo "<input type='hidden' name='users_id' value='$ID'>";
 
-         // All entities "edited user" have access
-         $strict_entities = Profile_User::getUserEntities($ID, true);
-         $user_entities = $_SESSION['glpiactiveentities'];
-         // Keep only entities "connected user" have access
-         $strict_entities = array_intersect($strict_entities, $user_entities);
-
-         $nb = countElementsInTableForEntity("glpi_groups", $strict_entities, '`is_usergroup`', false);
-         if ($nb > count($used)) {
-            Group::dropdown(array('entity'    => $strict_entities,
-                                  'used'      => $used,
-                                  'condition' => '`is_usergroup`'));
-            echo "</td><td>".__('Manager')."</td><td>";
-            Dropdown::showYesNo('is_manager');
-
-            echo "</td><td>".__('Delegatee')."</td><td>";
-            Dropdown::showYesNo('is_userdelegate');
-
-            echo "</td><td class='tab_bg_2 center'>";
-            echo "<input type='submit' name='addgroup' value=\""._sx('button','Add')."\"
-                   class='submit'>";
-
-         } else {
-            _e('None');
-         }
+         $params = ['used'      => $used,
+                    'condition' => "is_usergroup
```

Comment:
My first idea was to tell you to avoid using "hardcoded" SQL... But I took a look at the code, and this is currently not possible.
I took a look at the "condition" stuff from the dropdown, and I really don't like the way this is handled; I wonder if we can use a more reliable way.

Avoid using "condition" parameter would be the best option, or we'll probably need to rewrite that part :/